test_accounts/appealable_issues_test_accounts.md

```diff
@@ -10,7 +10,7 @@ The authentication model for the [Appealable Issues API](https://developer.va.go
 
 **To use this test data**
 1. Find the test accounts that meet your use case from the table below.
-2. Access login credentials for these accounts and instructions for using ID.me and Login.gov by following the link in the email that was sent to you when you signed up for sandbox access.  If you do not have this email, [request sandbox access](https://developer.va.gov/explore/api/benefits-claims/sandbox-access) on developer.va.gov.
+2. Access login credentials for these accounts and instructions for using ID.me and Login.gov by following the link in the email that was sent to you when you signed up for sandbox access.  If you do not have this email, [request sandbox access](https://developer.va.gov/explore/api/appealable-issues/sandbox-access) on developer.va.gov.
```

Comment:
This URL isn't live on the prod developer portal yet, but it will be when we make the API publicly available. It can be previewed [on staging](https://staging-developer.va.gov/explore/api/appealable-issues/sandbox-access).